recipes/abseil/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.2.0)
+project(test_package CXX)
+
+# We set it only for the convenience of calling the executable
+# in the package test function
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(absl REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} main.cpp)
+# Note: Conan 1.21 doesn't support granular target generation yet.
+target_link_libraries(${PROJECT_NAME} absl::absl)
```

Comment:
```suggestion
target_link_libraries(${PROJECT_NAME} absl::absl)
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 14)
```

Reply:
Why we need it here? Abseil only breaks behaviour in the C++17 If I'm not missing anything.

Reply:
I see, as you are checking the default cppstd version, it's solved.